constellations/gemini-admin/src/ask_for_help/afh.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+require("./afh.sass");
+import React from "react";
+import moment from "moment";
+import API from "../api.js";
+import { Link } from "react-router-dom";
+
+export class AskForHelpPage extends React.Component {
+    state = {
+        list: [],
+    };
+
+    componentDidMount() {
+        /*let data = {
+            id: 1,
+            title: "AP Calculus Help",
+            date: new Date(),
+            timeString: "2:00-4:00PM",
+            subject: "AP Calculus",
+            locationId: "wchs",
+            notes: "test note"
+        }
+        API.post("api/askforhelp/create", data).catch((err) => alert(err))*/
```

Comment:
Did you try making more than one fake data row to see what it would look like?

Reply:
Yeah, I left it there in the beginning so I could copy the code over to the edit page later, I guess I forgot to remove it